### PR TITLE
NEW Add extra information on full deploys.

### PR DESCRIPTION
**Pull request by mateusz**

### CODE REVIEW

File: code/model/jobs/DNDeployment.php [hunk 1]
```diff
@@ -197,6 +197,66 @@ public function getTags() {
...
+			default: return 'info';
+		}
+	}
+
+	/**
+	 * Convert status to a friendly form, shown on the frontend.
+	 *
+	 * @return string
+	 */
+	public function getStatusFriendly() {
```
Comment (halkyon) [inline code/model/jobs/DNDeployment.php]: This isn't needed, as you can just use `ResqueStatus` in the template which remaps "Started" to "Running" as well.

Comment (mateusz) [inline code/model/jobs/DNDeployment.php]: I removed the code because it doesn't belong in this commit :P ready for re-review.
